src/components/CreateVideo.js

```diff
@@ -33,14 +33,33 @@ const CreateVideo = props => {
 
   const [titleLoading, setTitleLoading] = useState(false);
 
+  const [optionsClicked, setOptionsClicked] = useState(false);
+
+  const [jitHover, setJitHover] = useState(false);
+  const [deepHover, setDeepHover] = useState(false);
+  const [truncateHover, setTruncateHover] = useState(false);
+  const [pitchHover, setPitchHover] = useState(false);
+  const [tempoHover, setTempoHover] = useState(false);
+  const [smoothHover, setSmoothHover] = useState(false);
```

Comment:
This is all kinda wild how you're doing this

Reply:
Looking at all these holds and am realizing that a custom hook could've done the work. I may have to refactor some stuff.